code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -497,7 +497,14 @@ emp_act
 			return 0
 	..()
 
+/mob/living/carbon/human/fire_act()

```

Comment:
So people can no longer turn themselves into blazing infernos and be immune to it?

Yeah, I see no reason to remove this at all. It's fun and *cool*.